### ISSUE TITLE
node code dumps while transcoding

### ISSUE THREAD

**Issue by gjannece** [HaveAGitGat/Tdarr#266]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by HaveAGitGat** [comment]
What do you mean by this? If a transcode fails then the error log output can be viewed by pressing the 'i' button on the transcode error tab.

**Comment by gjannece** [comment]
Ok, I figured it out. 
So ffmpeg wouldn't coredump, but nodejs did. I was using Ubuntu 18.04-LTS and had used nvm to install nodejs. I was using the correct version (8.16.1) for my user, but the program runs as root and root was using the wrong version (8.10.1). Once I changed the nodejs version, the coredumps stopped. I know it is in the setup instructions to run this with a certain nodejs version, but maybe adding a node version check/warning to see if the correct / supported version is being used. 
Thanks for making this! You have saved me huge amounts of harddrive space.
